packages/hooks/src/start.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+import { spawn } from 'child_process';
+import path from 'path';
+import fs from 'fs';
+
+const { main: pkgJSONMain } = JSON.parse(fs.readFileSync('./package.json', 'utf-8'));
+
+/**
+ * Start hook implementation that verifies and runs an app in Socket Mode.
+ * @param {string} cwd - The current working directory of the project.
+ * @param {string | undefined} customPath - An optional path for the app.
+ */
+(function _(cwd, customPath) {
+  // TODO - Format so that its less miss-able in output
+  console.log('Preparing local run in developer mode (Socket Mode)'); // eslint-disable-line no-console
+  validateEnvironment();
+
+  // tries the provided path, then package.json main, then defaults to index.js in the current
+  // working directory
+  const pkgJSONDefault = 'index.js';
+  const fullPath = path.resolve(cwd, customPath || (pkgJSONMain || pkgJSONDefault));
+  console.log(fullPath); // eslint-disable-line no-console
+
+  // Kick off a subprocess to run the app in development mode
+  const app = spawn('node', [`${fullPath}`]);
+  app.stdout.setEncoding('utf-8');
+  app.stdout.on('data', (data) => {
+    process.stdout.write(data);
+  });
+  app.stderr.on('data', (data) => {
+    process.stderr.write(data);
+  });
+
+  app.on('close', (code) => {
+    console.log(`bolt-app local run exited with code ${code}`); // eslint-disable-line no-console
```

Comment:
```suggestion
    console.log(`Local run exited with code ${code}`); // eslint-disable-line no-console
```

Reply:
I'm in favor of keeping this log though! It only seems to appear if the app crashes which is pretty useful. And this is a nice change! Thanks 😄